tests/dm_control/test_dm_control_policy.py

```diff
@@ -0,0 +1,57 @@
+import gc
+
+from dm_control.suite import ALL_TASKS
+from nose2 import tools
+
+from garage.baselines import LinearFeatureBaseline
+from garage.envs.dm_control import DmControlEnv
+from garage.tf.algos import TRPO
+from garage.tf.envs import TfEnv
+from garage.tf.policies import CategoricalMLPPolicy, GaussianMLPPolicy
+from garage.tf.spaces import Box
+from tests.fixtures import GarageTestCase
+
+all_tasks_list = []
+for task in ALL_TASKS[0:1]:
```

Comment:
What does this do exactly? Why can't we pass ALL_TASKS directly to `@tools.params`?

Reply:
Because it's in tuple form, I wasn't able to pass it directly as ALL_TASKS

Reply:
don't remember the exact error though